etcdctl/ctlv3/command/make_mirror_command.go

```diff
@@ -45,7 +48,8 @@ func NewMakeMirrorCommand() *cobra.Command {
 	}
 
 	c.Flags().StringVar(&mmprefix, "prefix", "", "Key-value prefix to mirror")
-	// TODO: add dest-prefix to mirror a prefix to a different prefix in the destination cluster?
+	c.Flags().StringVar(&mmdestprefix, "dest-prefix", "", "destination prefix to mirror a prefix to a different prefix in the destination cluster")
+	c.Flags().BoolVar(&mmremdestprefix, "no-dest-prefix", false, "mirror key-values to the root of the destination cluster")
```

Comment:
I don't understand why this option is necessary. Doesn't `--dest-prefix=""` cover this case?